lms/djangoapps/student_account/views.py

```diff
@@ -592,3 +596,45 @@ def account_settings_context(request):
         } for state in auth_states if state.provider.display_for_login or state.has_account]
 
     return context
+
+
+class RecoverPasswordView(ViewSet):
+
+    """
+    **Use Cases**
+            Resets a password of a user an email address
```

Comment:
I would call this a nit but I really couldn't understand this sentence ._. Please re-read and you will know why :P